data/team2020clinical/team2020clinical-extraction.md

```diff
@@ -0,0 +1,89 @@
+# Extraction for The COVID-19 Investigation Team (2020)
+
+[The COVID-19 Investigation Team (2020)](https://www.nature.com/articles/s41591-020-0877-5) studied clinical and virologic characteristics of the first 12 patients with coronavirus disease 2019 (COVID-19) in the United States. Respiratory, stool, serum, and urine specimens were submitted for SARS-CoV-2 real-time reverse-transcription polymerase chain reaction (rRT-PCR) testing, viral culture, and whole genome sequencing. Only nasopharyngeal samples were included in this dataset. Data for the nasopharyngeal swab results were obtained from the combined dataset in the supplementary materials of [Challenger et al. (2022)](https://doi.org/10.1186/s12916-021-02220-0). Attributes of hospitalized patients were obtained from the original paper. 
+
+First, we `import` python modules needed:
+
+```python
+#import modules;
+import yaml
+import pandas as pd
+import numpy as np
```

Comment:
`numpy` was not used in the code. Sorry, I added this in an earlier development version and you guys must inherit it from my codes for Liu2024. I will submit a commit to remove this.